This snippet includes an intentional error, so you can test that everything is working as soon as you set it up.

```typescript
app.get("/debug-sentry", async (c) => {
  throw new Error("My first Sentry error!");
});
```

To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and select your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
